test/In.ProjectEKA.HipServiceTest/OpenMrs/HealthCheck/HealthCheckMiddlewareTest.cs

```diff
@@ -0,0 +1,122 @@
+namespace In.ProjectEKA.HipServiceTest.OpenMrs
+{
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading.Tasks;
+    using System;
+    using FluentAssertions;
+    using In.ProjectEKA.HipService.OpenMrs.HealthCheck;
+    using Microsoft.AspNetCore.Http;
+    using Moq;
+    using Newtonsoft.Json;
+    using Xunit;
+
+    [Collection("Health Check Middleware Tests")]
+    public class HealthCheckMiddlewareTest
+    {
+
+        private Mock<IHealthCheckStatus> healthCheckStatus;
+        private Mock<IHealthCheckClient> healthCheckClient;
```

Comment:
Codacy found an issue: [Remove the field 'healthCheckClient' and declare it as a local variable in the relevant methods.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6377136)